src/api/enumeration.js

```diff
@@ -0,0 +1,64 @@
+import config from './util/config';
+import axios_cancelable from './util/axiosCancel';
+import setHeaders from './util/setHeaders';
+import { prettifyCountry, prettifyUserType } from './util/prettifiers';
+
+export const getCountries = () => {
+  return axios_cancelable.get(`${config.dataApi}/enumeration/basic/Country`, {
+    headers: setHeaders()
+  }).then(response => {
+    return response.data.map(code => ({
+      code,
+      name: prettifyCountry(code)
+    }));
+  });
+};
+
+export const getContactTypes = () => {
+  return axios_cancelable.get(`${config.dataApi}/enumeration/basic/ContactType`, {
```

Comment:
Plain Axios for these is probably easier to read? It is not something that will ever be cancelled anyhow

Reply:
Sure, I'll replace it with just axios.